README.md

```diff
@@ -84,6 +84,21 @@ Definition YAML files currently created and tested:
 * ECMWF Radiosonde
 * ECMWF Aircraft
 
+The current ODB library (called ODB API) only supports Python 2.7.
+ECMWF will be releasing a new ODB library (called ODC) soon, that will support Python 3.
+Our expectation is that ODC will show up in the next few weeks.
+
+Until ODC arrives, we have to use python 2.7 for the ODB test and file conversion.
+
+The ODB file conversion test will be disabled by default so that developers can continue to work in Python 3.
+The ODB coding norms test will always be enabled.
+
+When developing the ODB code, you will need to work inside the container (Singularity or CharlieCloud) and with Python 2.7. 
+To enable the ODB file conversion test, add the ENABLE_ODB_API option to ecbuild as follows:
+~~~~~~~~
+ecbuild -DENABLE_ODB_API=1 <other_ecbuild_options> <path_to_source_directory>
```

Comment:
`ENABLE_ODB_API` is not being used anywhere in this repo.  Is that correct?

Reply:
There is an ecbuild_add_option(FEATURE ODB_API ...) command added to the top-level CMakeLists.txt.  I'm not sure if this is ecbuild or cmake, but the convention is to use ENABLE_<name of option> to add the feature in, so that is where the ENABLE_ODB_API=1 comes from.